### PR TITLE
Use internal/gnoi/gnoi.KillProcess in tests

### PR DESCRIPTION
**Pull request by self-maurya**

### CODE REVIEW

**Review by rohit-rp — APPROVED** [review]

File: feature/experimental/policy/otg_tests/prefix_set_test/prefix_set_test.go [hunk 1]
```diff
@@ -136,7 +136,6 @@ func TestPrefixSetWithOCAgentRestart(t *testing.T) {
 	gnoi.KillProcess(t, dut, gnoi.OCAGENT, true)
 
 	v4PrefixSet = ds.GetOrCreatePrefixSet(tag3IPv4)
-	v4PrefixSet.SetMode(oc.PrefixSet_Mode_IPV4)
```
Comment (dplore) [inline feature/experimental/policy/otg_tests/prefix_set_test/prefix_set_test.go:139]: why was this removed?  It seems unrelated to the main topic of this PR?

File: feature/experimental/bgp/ate_tests/bgp_long_lived_graceful_restart/bgp_long_lived_graceful_restart_test.go [hunk 2]
```diff
@@ -943,7 +885,7 @@ func TestTrafficWithGracefulRestartLLGR(t *testing.T) {
 		})
 
 		t.Run("Restart routing", func(t *testing.T) {
-			restartRoutingProcess(t, dut)
+			gnoi.KillProcess(t, dut, gnoi.ROUTING, true)
```
Comment (dplore) [inline feature/experimental/bgp/ate_tests/bgp_long_lived_graceful_restart/bgp_long_lived_graceful_restart_test.go:888]: So much cleaner!  Thank you.

File: feature/experimental/policy/otg_tests/prefix_set_test/prefix_set_test.go (hunk 1 above)
Comment (self-maurya) [reply feature/experimental/policy/otg_tests/prefix_set_test/prefix_set_test.go:139]: it was a miss in previous PR, this line is already covered in deviations in next few lines
